Source/WebCore/page/CaptionUserPreferences.cpp

```diff
@@ -272,7 +273,7 @@ Vector<RefPtr<TextTrack>> CaptionUserPreferences::sortedTrackListForMenu(TextTra
 
     Collator collator;
 
-    std::sort(tracksForMenu.begin(), tracksForMenu.end(), [&] (auto& a, auto& b) {
+    std::ranges::sort(tracksForMenu, [&] (auto& a, auto& b) {
```

Comment:
```suggestion
    std::ranges::sort(tracksForMenu, [&](auto& a, auto& b) {
```

Reply:
Given how often this specific comparison function is used, perhaps adding a TrackDisplayNameLessThan (or whatever one would want to call it) object to reuse would be useful.